frontend/router/index.tsx

```diff
@@ -114,9 +112,7 @@ const routes = (
             <Route path="windows" component={DashboardPage} />
           </Route>
           <Route path="settings" component={AuthAnyAdminRoutes}>
-            <IndexRedirect
-              to={isSandboxMode ? "integrations" : "organization"}
-            />
```

Comment:
Do we no longer need to redirect to a different path for sandbox users?

Reply:
This will by default route to the integrations sub-nav, which is visible in both sandbox and regular modes, so we don't. Ideally, non-sandbox routes to organization settings, but...

The issue stems from the fact that "isSandboxMode" isn't accessible at this level, so we have to figure out how to determine that at this point. Any ideas?

Reply:
Either way, this will work reliably in the meantime, if you think it's acceptable to default to the Settings > Integrations subnav.

Reply:
I don't think it's acceptable to default to Settings > Integrations. Let's see if we can find an alternate fix.

Reply:
Roger, trying another idea